.github/workflows/build-docker-image.yml

```diff
@@ -65,6 +65,7 @@ jobs:
           echo "NEXT_PUBLIC_DMOB_API_URL='${{ vars.NEXT_PUBLIC_DMOB_API_URL }}'" >> .env
           echo "NEXT_PUBLIC_GLIF_URL='${{ vars.NEXT_PUBLIC_GLIF_URL }}'" >> .env
           echo "NEXT_PUBLIC_IS_TESTNET='${{ vars.NEXT_PUBLIC_IS_TESTNET }}'" >> .env
+          echo "NEXT_PUBLIC_GOOGLE_ANALYTICS_TRACKING_ID='${{ vars.NEXT_PUBLIC_GOOGLE_ANALYTICS_TRACKING_ID }}'" >> .env
```

Comment:
Did we already add it to repo vars? If not, let's do it even before we merge.

Reply:
done